### PR TITLE
Textual Beta 6 (new formula)

### PR DESCRIPTION
**Pull request by jedahan**
- [x] Checked there aren’t open [pull requests](https://github.com/caskroom/homebrew-versions/pulls) for the same cask.
- [x] Checked there aren’t closed [issues](https://github.com/caskroom/homebrew-versions/issues) where that cask was already refused.
- [x] Checked the cask follows the requirements of [acceptable casks](https://github.com/caskroom/homebrew-versions#acceptable-casks).
- [x] When naming the cask, followed the [token reference](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [x] Commit message includes cask’s name.
- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` left no offenses.
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.

### CODE REVIEW

File: Casks/textual-beta.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+cask 'textual-beta' do
+  version '6.0.0'
```
Comment (vitorgalvao) [inline Casks/textual-beta.rb]: Seeing as `url` doesn’t point to a specific version, this cask should use `version :latest` and `sha256 :no_check`. [See an example](https://github.com/caskroom/homebrew-cask/blob/f25b6babcd398abf48e33af3d887b2d00de1d661/Casks/kibako.rb).

**Comment by jedahan** [comment]
ok, made the change

File: Casks/textual-beta.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
+cask 'textual-beta' do
+  version :latest
+  sha256 :no_check
+
+  url 'https://www.codeux.com/textual/downloads/Textual-Beta.zip'
+  name 'Textual'
+  homepage 'https://www.codeux.com/textual/'
```
Comment (vitorgalvao) [inline Casks/textual-beta.rb]: Is there a more specific page, for the beta?

Comment (vitorgalvao) [inline Casks/textual-beta.rb]: Could you please answer this?

**Comment by jedahan** [comment]
https://help.codeux.com/textual/Release-Notes:-Version-6.0.0.kb was the most textual 6 specific page I could find

**Pull request merged**